openmindat/geomaterials.py

```diff
@@ -1394,14 +1421,39 @@ def save(self, FILE_NAME = ''):
 
             Example:
                 >>> gir = GeomaterialIdRetriever()
-                >>> gir.save()
+                >>> gir.id(5).save()
         '''
         file_name = FILE_NAME
         
         self.saveto('', file_name)
         
+    def get_dict(self):
```

Comment:
I know this function is currently not working, but it is also recommended to modify it into get_list for better consistency. Users will learn that they can get the item object by calling `get_list` from any endpoint without trying different functions.